job_spec/ARIA_S1_GUNW.yml

```diff
@@ -0,0 +1,86 @@
+ARIA_S1_GUNW:
+  required_parameters:
+    - granules
+    - secondary_granules
```

Comment:
I wonder if we should call these `reference` and `secondary` to match the upcoming [INSAR_ISCE_MULTI_BURST](https://github.com/ASFHyP3/hyp3/blob/develop/job_spec/INSAR_ISCE_MULTI_BURST.yml) job type.

Reply:
Also to match the `--reference-scenes` and `--secondary-scenes` CLI params.